src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -81,6 +81,13 @@ public string GetVersion()
             return header == null ? null : header.Value.ToString();
         }
 
+        ServiceControlRootUrls GetRootUrls()
```

Comment:
@Particular/serviceinsight-maintainers this API could be used to replace all the hardcoded URIs in the future. We figured that is something that can be done at a later stage as a refactoring